pallets/dkg-metadata/src/mock.rs

```diff
@@ -126,14 +128,32 @@ where
 	}
 }
 
+pub const MILLISECS_PER_BLOCK: u64 = 10000;
```

Comment:
Earlier, I left a comment about whether or not we need 64 bits to store the number of milliseconds. If we don't want to have to change the types here just to accommodate that, it's not a big deal.

Reply:
I think let's leave this as is currently.